src/pyze/cli/charge-history.py

```diff
@@ -40,6 +40,8 @@ def run(parsed_args):
                 'Duration',
                 'Power (kW)',
                 'Charge gained (%)',
+                'Started at (%)',
```

Comment:
It feels more logical to have "Started at" before "charge gained". Please could you switch the order?